src/Components/Catalog/Header/Header.js

```diff
@@ -1,13 +1,35 @@
+import { Link , useLocation} from 'react-router-dom';
 import style from '../../../styles/Header.module.css'
 
 const Header = () => {
+
+    const location = useLocation();
+
+    let links = [];
+
+    if (location.pathname === '/') {
+        links = [
+        { to: '#', label: 'Search' },
+        { to: '#', label: 'Create' },
+        ];
+    } else {
+        links = [
+        { to: '/', label: 'Main' }
+        ];
+    }
+
```

Comment:
avoid such structures. You can just create array with all links and render them below as `{location.pathname === '/' && <Link to="#">Search</Link>}`. Also it will be convinient to create separate variable for this condition `location.pathname === '/' `. We can discuss it tomorrow on the meeting

Reply:
done